packages/parser/src/validators/schema.ts

```diff
@@ -76,7 +76,6 @@ export function validateSchema(
       delete schemaDynamicRef.$dynamicAnchor;
     }
 
-    /* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
thoughts on adding [this rule](https://biomejs.dev/linter/rules/no-ts-ignore/) to our shared config? i am not a fan of `@ts-ignore`